app/jobs/malware_scan_job.rb

```diff
@@ -0,0 +1,22 @@
+class MalwareScanJob < ApplicationJob
+  RETRY_WAIT_TIME = 5.minutes
+
+  retry_on Faraday::ServerError, wait: RETRY_WAIT_TIME, attempts: :infinite
+  retry_on Faraday::ConnectionFailed, wait: RETRY_WAIT_TIME, attempts: :infinite
+
+  def perform(attachment_id)
+    attachment = ActiveStorage::Attachment.find(attachment_id)
+
+    return if attachment&.blob.blank?
```

Comment:
```suggestion
    return if attachment.blob.blank?
```

`find` lève déjà un `ActiveRecord::NotFoundError`

Il faudrait le catch, ou alors modifier le code pour faire un `find_by` et `return if attachment.blank? || attachment.blob.blank?` (qui est plus clair)